src/diffusion_for_multi_scale_molecular_dynamics/generators/position_generator.py

```diff
@@ -11,6 +11,7 @@ class SamplingParameters:
 
     algorithm: str
     spatial_dimension: int = 3  # the dimension of Euclidean space where atoms live.
+    num_atom_types: int = 3  # number of atom types excluding MASK
```

Comment:
I don't think there should be a default value. This will lead to subtle bugs / conflicts for inexperienced users. The user should be forced to specify this in the config file (or something).